test/cql/lwt_batch_test.cql

```diff
@@ -1,3 +1,8 @@
+CREATE KEYSPACE ks
```

Comment:
If not this explicit `ks` creation, which keyspace would the below tests use?

Reply:
the pytest generates a keyspace with a random name

Reply:
OK. Since you use not so random name for vnodes keyspace here, I'd suggest that test.py generates two keyspaces with fixed names, say `ks_vnodes` and `ks_tablets` so that each .cql file could just `use $ks_they_want`

Reply:
A random name is desired for some tests, if a test leaves around a table, this may impact a subsequent test, and the problem may not be visible in a dev run or some CI runs, since we can't expect a fixed order of test files during execution - files get added, disabled, etc.

So not using a random name is extra responsibility on the test writer.

Second, it may seem when looking at this patch that these two keyspaces would be enough. In fact we may have other non-default options, required for some tests. 

The pull request is trying to solve a specific problem with the means that are already around, and are already used in several tests.

Reply:
@mykaul some existing tests in cql/ tests explicitly avoid random names because they show up in error messages returned by the server, and this breaks approval  .result files.

Reply:
Yes, that's why it's semi-random - there's the testname in them, so you can search it with a regexp (ignoring the random bits)

Reply:
@mykaul this is not how approval test work. The output of the server is written verbatim into the result file, and if it differs even 1 bit, the test fails.

Reply:
Let me summarize:
Approval tests create a random keyspace already for most of the tests; this works for most of the tests; the keyspace is dropped automatically at the end of the test. This is a convenience, and it's totally fine to create a separate keyspace in the test if necessary. This is nearly instant since these are single-node tests.

Some tests indeed use custom keyspace.  Until this patch the only reason was to stabilize test output.  After that, it is also to make the test stable in the face of the changing system-wide keyspace defaults.